arangod/Cluster/EnsureIndex.cpp

```diff
@@ -116,6 +117,27 @@ bool EnsureIndex::first() {
       body.add(VPackObjectIterator(props));
     }
 
+    uint64_t docCount = 0;
+    if (arangodb::maintenance::collectionCount(*col, docCount).fail()) {
+      std::stringstream error;
+      error << "failed to get count of local collection " << shard << " in database " + database;
+      LOG_TOPIC("23561", ERR, Logger::MAINTENANCE) << "EnsureIndex: " << error.str();
+      result(TRI_ERROR_INTERNAL, error.str());
```

Comment:
Here I would prefer to not dismiss the actual error, but keep it.
I also suggest to downgrade the log level from ERR to WARN, because the issue may fix itself in a following run.
```suggestion
    if (Result res = arangodb::maintenance::collectionCount(*col, docCount); res.fail()) {
      std::stringstream error;
      error << "failed to get count of local collection " << shard << " in database " << database << ": " << res.errorMessage();
      LOG_TOPIC("23561", WARN, Logger::MAINTENANCE) << "EnsureIndex: " << error.str();
      result(res.errorNumber(), error.str());      
```